.sourcery.yaml

```diff
@@ -0,0 +1,31 @@
+---
+rule_settings:
+  disable: [replace-interpolation-with-fstring]
+  python_version: '3.6'
```

Comment:
The docs are opaque about what python_version means. Does this only apply to 3.6?

Reply:
It ensures that the proposed changes are compatible with version no higher than specified.